web-build/Dockerfile.platformsh

```diff
@@ -3,3 +3,7 @@
 # platform self:update fails if there are no updates, and also fails if no internet
 RUN platform self:update -y --no-major || true
 RUN ln -sf /var/www/html /app
+RUN echo "deb-src http://deb.debian.org/debian bullseye main" >> /etc/apt/sources.list
+RUN apt-get update && apt-get -y build-dep python2 python3
+# python2-pip is no longer available for Debian bullseye
+RUN wget https://bootstrap.pypa.io/pip/2.7/get-pip.py && python get-pip.py
```

Comment:
Overall I just hate to add this much to the local image just in case they need python2. But we don't know what all they may be doing with python2.